File: 프로그래머스/0/181912. 배열 만들기 5/배열 만들기 5.cs
using System;
using System.Collections.Generic;

public class Solution 
{
    public int[] solution(string[] intStrs, int k, int s, int l) 
    {
        string strSub = "";
        List<int> numbers = new List<int>();
        for (int i=0; i<intStrs.Length; i++) {
            string str = intStrs[i];
            for(int j=s; j<s+l; j++) {
                strSub += str[j].ToString();
            }
            int num = Int32.Parse(strSub);
            if (num > k) {
                numbers.Add(num);   
            }
            strSub = "";
        }
        int[] answer = numbers.ToArray();
        return answer;
    }
}

File: 프로그래머스/0/181912. 배열 만들기 5/배열 만들기 5.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<string> intStrs, int k, int s, int l) 
{
    vector<int> result;
    for(int i=0; i<intStrs.size(); i++)
    { 
        string subStr = intStrs[i].substr(s,l);
        if(stoi(subStr) > k)
        {
            result.push_back(stoi(subStr));
        }
    }
    return result;    
}